tests/fixtures/zwave_js/binary_switch_state.json

```diff
@@ -0,0 +1,727 @@
+{
```

Comment:
Please name the fixture file after the device so we keep track of what fixtures we have.